### PR TITLE
[SHELL32] Improve French (fr-FR) translation

### PR DESCRIPTION
**Pull request by fulup29**
[SHELL32] Fix french translation and layout

original patch by ghost,  pull request 4710

JIRA issue: [CORE-18382](https://jira.reactos.org/browse/CORE-18382)

Original patch is used to fix and improve french translation and layout

Before
before

After

with offset

### CODE REVIEW

**Comment by KRosUser2** [comment]
Screenshot before/after please

**Comment by KRosUser2** [comment]
> The different x-coordinates for the 2 AUTOCHECKBOXES... Does that affect any other language as well?

Should it be adressed in this specific PR ?

**Comment by KRosUser2** [comment]
@JoachimHenze : what's your recommendation?

File: dll/win32/shell32/lang/fr-FR.rc [hunk 1]
```diff
@@ -609,17 +609,17 @@ BEGIN
...
-    GROUPBOX "Options de vérification", -1, 7, 6, 179, 40
-    PUSHBUTTON "Annuler", IDCANCEL, 128, 95, 60, 14
-    AUTOCHECKBOX "Réparer automatiquement les erreurs du système de fichiers", 14000, 12, 15, 155, 10
-    AUTOCHECKBOX "&Scanner pour rechercher et réparer les secteurs défectueux", 14001, 16, 30, 165, 10
-    CONTROL "", 14002, "MSCTLS_PROGRESS32", 16, 7, 50, 179, 8
+    DEFPUSHBUTTON "Démarrer", IDOK, 113, 95, 60, 14
+    GROUPBOX "Options de vérification du disque", -1, 7, 6, 229, 40
+    PUSHBUTTON "Annuler", IDCANCEL, 178, 95, 60, 14
+    AUTOCHECKBOX "Réparer automatiquement les erreurs du système de fichiers", 14000, 12, 20, 215, 10
+    AUTOCHECKBOX "Rechercher et tenter une récupération des secteurs défectueux", 14001, 12, 30, 215, 10
```
Comment (HBelusca) [inline dll/win32/shell32/lang/fr-FR.rc:621]:
```suggestion
    AUTOCHECKBOX "&Réparer automatiquement les erreurs du système de fichiers", 14000, 12, 20, 215, 10
    AUTOCHECKBOX "Re&chercher et tenter une récupération des secteurs défectueux", 14001, 12, 30, 215, 10
```

**Comment by HBelusca** [comment]
> > The different x-coordinates for the 2 AUTOCHECKBOXES... Does that affect any other language as well?
> 
> Yes other lang (i did a quick sampling) have a 4px offset on the second checkbox Should it be adressed in this specific PR ? Or should the offset be kept here ? (but this 2nd option is not "dependant" on the 1st one)

Personally when there is some fundamental replacement of items like that, I think it would be better to have it done consistently in all languages at once.

**Comment by KRosUser2** [comment]
> > > The different x-coordinates for the 2 AUTOCHECKBOXES... Does that affect any other language as well?
> > 
> > 
> > Yes other lang (i did a quick sampling) have a 4px offset on the second checkbox Should it be adressed in this specific PR ? Or should the offset be kept here ? (but this 2nd option is not "dependant" on the 1st one)
> 
> Personally when there is some fundamental replacement of items like that, I think it would be better to have it done consistently in all languages at once. The results can be verified afterwards by opening the compiled DLL in any resource editor and observing the resulting dialog(s).

@fulup29 : Would you please restore the 4 px offset then ? (with new screenshot please)

File: dll/win32/shell32/lang/fr-FR.rc (hunk 1 above)
Comment (KRosUser2) [inline dll/win32/shell32/lang/fr-FR.rc:621]: Please restore 4px offset (see comment from @HBelusca )

**Review by KRosUser2 — CHANGES_REQUESTED** [review]

File: dll/win32/shell32/lang/fr-FR.rc (hunk 1 above)
Comment (HBelusca) [reply dll/win32/shell32/lang/fr-FR.rc:621]: either that, or remove that buggy offset in all translations.

**Comment by KRosUser2** [comment]
Would you please update the "post change" screenshot with the restored offset ?

**Comment by fulup29** [comment]
with offset
With offset

**Comment by HBelusca** [comment]
> With offset

Of course, with erroneous offset it looks like garbage.
I may invite you to make a separate PR that just fixes the offset in all languages at once.

File: CMakeLists.txt [hunk 2]
```diff
@@ -368,6 +368,7 @@ Enable this if the module uses typeid or dynamic_cast. You will probably need to
     add_subdirectory(sdk/tools/wpp)
     add_subdirectory(win32ss)
 
+
```
Comment (HBelusca) [inline CMakeLists.txt:371]:
```suggestion
```

**Review by HBelusca — APPROVED** [review]

File: dll/win32/shell32/lang/fr-FR.rc [hunk 3]
```diff
@@ -609,17 +609,17 @@ BEGIN
...
-    GROUPBOX "Options de vérification", -1, 7, 6, 179, 40
-    PUSHBUTTON "Annuler", IDCANCEL, 128, 95, 60, 14
-    AUTOCHECKBOX "Réparer automatiquement les erreurs du système de fichiers", 14000, 12, 15, 155, 10
-    AUTOCHECKBOX "&Scanner pour rechercher et réparer les secteurs défectueux", 14001, 16, 30, 165, 10
-    CONTROL "", 14002, "MSCTLS_PROGRESS32", 16, 7, 50, 179, 8
+    DEFPUSHBUTTON "Démarrer", IDOK, 113, 95, 60, 14
+    GROUPBOX "Options de vérification du disque", -1, 7, 6, 229, 40
+    PUSHBUTTON "Annuler", IDCANCEL, 178, 95, 60, 14
+    AUTOCHECKBOX "&Réparer automatiquement les erreurs du système de fichiers", 14000, 12, 20, 215, 10
+    AUTOCHECKBOX "Re&chercher et tenter une récupération des secteurs défectueux", 14001, 12, 30, 215, 10
```
Comment (HBelusca) [inline dll/win32/shell32/lang/fr-FR.rc:621]:
```suggestion
    AUTOCHECKBOX "Re&chercher et tenter une récupération des secteurs défectueux", 14001, 16, 30, 215, 10
```

**Comment by HBelusca** [comment]
@fulup29 For you other PRs you need to create them from different branches, instead of using the single master branch. Otherwise you will end up with mixed unrelated changes.

**Comment by KRosUser2** [comment]
Thank you @fulup29 and @HBelusca !

**Pull request merged**